mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js

```diff
@@ -421,9 +573,10 @@
       $typeMenu.attr("role", "menu");
 
       for ( var type in SearchEntity.TYPES) {
-        if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false)
-          continue;
-
+      	if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false || type.includes("_FALLBACK")) {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640)